src/dev/reg_bank.hh

```diff
@@ -861,9 +863,26 @@ class RegisterBank : public RegisterBankBase
         void reset() override { _resetter(*this); }
     };
 
+    // Allow gem5 models to set a debug flag to the register bank for logging
+    // all full/partial read/write access to the registers. The register bank
+    // would not log if the flag is not set.
+    //
+    // The debug flag is the one declared in the SConscript
+    //
+    // DebugFlag('HelloExample')
+    //
+    // Then the flag can be set in the register bank with:
+    //
+    // setDebugFlag(::gem5::debug::HelloExample)
+    void setDebugFlag(const ::gem5::debug::SimpleFlag& flag)
```

Comment:
Nit: void should be on a separate line:

```
void
setDebugFlags
```

Reply:
Bump

Reply:
Sorry for the late reply. I have fixed this part.